lib/sign_in/idme/service.rb

```diff
@@ -49,12 +49,18 @@ def user_info(token)
       private
 
       def get_public_jwks
-        unless config.public_jwks
-          response = perform(:get, config.public_jwks_path, nil, nil)
-          config.public_jwks = parse_public_jwks(response:)
-          Rails.logger.info('[SignIn][Idme][Service] Get Public JWKs Success')
+        return config.public_jwks if config.public_jwks
```

Comment:
Probably a nit pick, but it seems like a few of the methods in the idme and logingov classes are repeated and could maybe benefit from a helper module. 🤷‍♀️ Just a thought for a cleanup item for next time, but otherwise LGTM